Makefile

```diff
@@ -37,7 +37,7 @@ COCKPIT_REPO_FILES = \
 	$(NULL)
 
 COCKPIT_REPO_URL = https://github.com/cockpit-project/cockpit.git
-COCKPIT_REPO_COMMIT = 30eeca9eeabdf9a2f837151992d40b88d0f34cd3 # 290 + chromium 113 sizzle fix
+COCKPIT_REPO_COMMIT = 7d2f1890a578ed00572c514141094d9ca28b63b8 # 292 + 40 commits
```

Comment:
Can you please include at least 43c3606ed474ea0f841551d7cf9770f38c88a5aa to pick up the pybridge flake fix? There are no other changes which would affect c-certificates, so that should be safe.